### PR TITLE
decision logging: Improve error message when chunk size > limit 

### PR DESCRIPTION
**Pull request by anderseknert**
Fixes https://github.com/open-policy-agent/opa/issues/5155

### CODE REVIEW

File: docs/content/configuration.md [hunk 1]
```diff
@@ -306,8 +306,7 @@ If persistence is not configured the OCI downloader will store the layers in the
 OPA will authenticate using the specified bearer token and schema; to enable bearer token
 authentication, either the token or the path to the token must be specified. If the latter is provided, on each request OPA will re-read the token from the file and use that token for authentication.
 
-The schema is optional and will default to `Bearer`
-if unspecified.
+The `scheme` attribute is optional, and will default to `Bearer` if unspecified.
```
Comment (philipaconrad) [inline docs/content/configuration.md:309]: I like that this is clearer. Looks unrelated to this PR, but I'm fine rolling it in with the other fixes.

**Comment by philipaconrad** [comment]
Waiting for main test suite jobs to go green before greenlighting this PR.

Thanks for the improved error message, @anderseknert! :smiley:

**Review by philipaconrad — APPROVED** [review]
LGTM! :+1: Should be good merge when green.

File: docs/content/configuration.md (hunk 1 above)
Comment (anderseknert) [reply docs/content/configuration.md:309]: It's unrelated for sure, but did not feel like a separate PR was warranted :)

**Pull request merged**